decision-process/README.md

```diff
@@ -0,0 +1,134 @@
+- The YAML header must contain the following fields in this order:
+
+  - `creation_date: YYYY-MM-DD`
+  - `issues` containing links to GitHub issues. Can be null or an empty list. The issues can be the origin of the RFC or where the work for implementing the decision takes place.
+  - `owners` containing GitHub team URLs. This list must not be empty. Please enter only the team and/or SIG names of the author and owners, not the teams who should comment on the RFC (the _stakeholders_, for which we don't have a structured field). The `owners` field contains teams who have the expertise to check the RFC after a few months. It may later be used to automatically create GitHub issues "Please review if this RFC is still relevant" (see [Future ideas](#future-ideas)).
+  - `state` having one of the values
+
+    - `review` – The RFC pull request is in review and waiting for comments. The decision is not in effect.
+    - `approved` – The decision was approved and is in effect if the PR was merged.
+    - `declined` – The decision was not approved and will not be in effect, meaning the RFC is closed.
+    - `obsolete` – The RFC or decision was replaced by something newer (which should be linked in the Markdown content) or is just not relevant anymore (example: product or technology not used anymore).
+  - `summary` should summarize the RFC in 1-3 concise sentences. It is meant for people who want to get a very quick overview instead of reading whole RFC documents. That is, _most people_! We use this in the rendered version to provide a simple overview.
+- YAML headers will be checked by CI upon creation or push of a PR in the [`rfc` repo](https://github.com/giantswarm/rfc)
+
+Other than that, an RFC only requires a heading and otherwise remains free-form.
+
+The following is an entire and valid start of an RFC markdown file:
+
+```text
+---
+creation_date: 2023-07-13
+issues:
+  - https://github.com/giantswarm/giantswarm/issues/24661
+owners:
+  - https://github.com/orgs/giantswarm/teams/team-horizon
+state: review
+---
+
+# This is the RFC title
+```
+
+## Overview of RFCs by adding a list to the handbook
+
+We previously lacked an overview of RFCs. There was no way to sort RFCs by date, see which one is still relevant, or get a one-sentence summary in order to quickly skim through the decisions Giant Swarm has made in the past.
+
+Therefore, we will automatically render the merged RFCs into our [handbook](https://handbook.giantswarm.io/docs/) and provide at least a table as overview.
+
+## Decision making process
+
+- **Author opens pull request**
+
+  - Clone the [`rfc` repo](https://github.com/giantswarm/rfc)
+  - Create a new directory such as `nginx-ingress-app-migration`. It should be named like the others, i.e. without spaces. Mind that this name (so-called _slug_) is later used in the handbook URL of the rendered RFC and should not change.
+  - In that directory, create a `README.md` with the following template:
```

Comment:
We might create this as template file?

Reply:
For personal stuff, I have something like a `create-rfc.sh` script that asks certain information (date, slug, title) and creates the file accordingly. I had a similar idea, but was wondering whether we can make this whole procedure as un-technical as possible so that it's open to anyone. Is there a way to solve this by clicking around GitHub's UI, not requiring a local checkout of the repo, and using the WYSIWYG Markdown editor of GitHub? A template file sounds good if GitHub allows copying that, or has some way to get started writing an RFC without cloning the repo. But maybe I'm mistaken and git is not a hurdle – WDYT?

Reply:
I'm leaving the template here in the "RFC of RFCs" for now. As long as people use a correct header and h1 heading, it doesn't matter if the file got copy-pasted, came from a template or whatever. People will shout if they think the process is complicated or needs a simple script to create an RFC. We can improve on that when we see it necessary.